src/core/city-autocomplete.service.js

```diff
@@ -34,7 +35,7 @@ function CityAutocomplete($q, $http, appConfig, UdbPlace, jsonLDLangFilter) {
         'workflowStatus': 'DRAFT,READY_FOR_VALIDATION,APPROVED',
         'disableDefaultFilters': true,
         'embed': true,
-        'limit': 1000,
+        'limit': (asyncPlaceSuggestionsFeatureToggle) ? 1000 : 3000,
```

Comment:
The current max number of items per page on the backend is 2000: https://documentatie.uitdatabank.be/content/search_api_3/latest/getting-started/pagination.html

We could probably increase it to 3000 but I won't be able to do that until tomorrow.